tools/taskcluster/execute-bitbar-test.py

```diff
@@ -0,0 +1,52 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""
+This script initiates a UI test-run after a upload of both the application and test
+APKs to Bitbar Cloud.
+"""
+
+from testdroid import Testdroid
+import json
+import sys
+import time
+
+with open('../../.bitbar_token.json') as json_file:
+    token_data = json.load(json_file)
+
+testdroid = Testdroid(apikey=token_data['bitbarToken'], url=token_data['cloudURL'])
+
+if len(sys.argv) != 3:
+    print "Bitbar Cloud: Application and Test Application APK path required."
+    exit(1)
```

Comment:
This could just be me misunderstanding this API, but why does our log only list two required args when we're checking for the presence of 3?

Reply:
A little further down, it looks like args 1 and 2 both refer to the Application?  Is that correct?  Could we clarify what the required args are here?

Reply:
First is the script in Python